pages/1.11/release-notes/1.11.9/index.md

```diff
@@ -0,0 +1,122 @@
+- DCOS-46915 - In OTP 20 and earlier, all Erlang distribution protocol connections over TLS are initialized by `ssl_tls_dist_proxy` one at a time. This approach causes a bottleneck and is resolved in the newest OTP versions. If the node connects to a non-existing node, it takes up to 30 seconds to get an error. Lashup tries to connect to such nodes every `n` seconds which causes a message storm in `ssl_tls_dist_proxy`. The workaround is to restart the entire virtual machine, if there are more than `m` messages in the queue. It is recommended not to kill the `ssl_tls_dist_proxy` since it will break all new and old distribution protocol connections.
+- DCOS_OSS-3736 - Marathon randomly assigns host ports. In rare cases, it causes port conflicts in the networking integration tests. The fix is to assign unique port numbers manually and update Marathon app configuration to match Marathon 1.5 syntax.
+
+## Package Management
+- DCOS_OSS-4418 - The requests package before 2.20.0 for python sends a `HTTP` authorization header to a `HTTP URI` upon receiving a same host name `https-to-http` redirect, which makes it easier for remote attackers to discover credentials by sniffing the network. This causes a vulnerability in the requests library necessitating an upgrade. This release upgrades the version of the requests library to an updated and more secure version 2.20.1 and urllib3 to 1.24.1.
+
+[enterprise]
+## Security
+[/enterprise]
+- DCOS_OSS-4742 - Currently, if the `validate_success` or `validate_error` function fails during gen unit tests; the user is not aware of the full output on why the assert failed. This release adds proper `AssertionError` reporting to resolve this issue. 
+
+# Known Issues and Limitations
+This section covers any known issues or limitations that don’t necessarily affect all customers, but might require changes to your environment to address specific scenarios. The issues are grouped by feature, functional area, or component. Where applicable, issue descriptions include one or more issue tracking identifiers.
+
+## Mesos
+- DCOS-43987 - Restore accidently deleted tests.
+- DCOS-44935, DCOS_OSS_3877, DCOS_OSS-4658 - The diagnostics bundle is the standard way of collecting the debugging information from a cluster to debug and fix critical issues on customer sites. Most of the powerful debugging information is lost since the diagnostic bundle is not created as the root user and network information is not collected as part of the diagnostic bundle. To resolve this issue, create `dcos-diagnostics` as the root user which will collect the best possible diagnostic information in order to effectively debug issues.
+
+## Networking 
+- DCOS_OSS-4667 - Mesos recently introduced a flag to toggle CNI root directory to persist across reboot. Now, you can expose the Mesos flag through DC/OS config.yaml file.
```

Comment:
I am confused by the placement of this one in the "Known issues section", isn't this a fix?   It should under "Installation", I think.